### PR TITLE
adding hardware documentation #7

### PR DESCRIPTION
**Pull request by GG23800**

### CODE REVIEW

**Comment by Aurelie-Mutschler** [comment]
ping @Bivi could you please review the hardware documentation @jerome2echopen just made? we need some feedbacks, for example are there enough details or should we add some information ?

**Comment by Bivi** [comment]
For me the second diagram is confusing : in french, very similar to the first one and false (we never decided to control directly the TGC with a smartphone ...). It would be simpler and more usefull to choose only one diagram (even if the first one is functional and the second a design diag) and add one paragraph for each component containing all data concerning this component and links with others, . Example :

     ...
## TGC
Time Gain Compensation, is an electronic component used to compensate signal attenuation during time.  TGC output is theoretically in range +/- 2.5 V, generally it is between +/- 1V....bla bla bla. TGC amplifies signal from ... TGC is controlled by ... TGC output is used by ...
### TGC caracteristic:
Amplification command: between 0 and 1 V
Amplification on LO gain mode: from -4.5 to +43.5 dB
Amplification on HI gain mode: from +7.5 to +55.5 dB
Output: between -2.5 and +2.5 V
### TGC Settings
TGC: LO gain mode (less amplification but too much noise on HI gain mode)
TGC amplication command: between 0.3 and 1 V (so amplification is between 10 and 43.5 dB)
   ...

**Comment by Aurelie-Mutschler** [comment]
thanks @Bivi !
I wait for @jerome2echopen to update the file before I merge the PR. I'll have a look at it more in details by the beginning of next week.

**Comment by GG23800** [comment]
Yep, the synoptic diagram was wrong, I forgot to made all modifications needed.

I changed the order of the diagram with few description so one can look at one or the other. If you still think it is redundant, we will clear it.

I don't agree with the vision of describing all modules one by one like this. It is not the technical documentation of the hardware, it is a more general discussion about the device. And I think it preferable to put all the settings together so one don't have to navigate in the document to find all of them.

I make a new pull request.

File: stable/characteristics.md
Comment (Aurelie-Mutschler) [inline stable/characteristics.md]: I don't understand how we deduce the number of points per line from the previous sentence ?! The number of points is just related to the listening time and sampling rate, right? Why is the number of points constant for all depth values ? Does that mean that you change the sampling rate ?!

File: stable/characteristics.md
Comment (Aurelie-Mutschler) [inline stable/characteristics.md]: The information about image size  is relevant here, but the description of the data transfer between device and terminal should be in doc_hardware.md

File: stable/characteristics.md
Comment (Aurelie-Mutschler) [inline stable/characteristics.md]: Similarly, this paragraph is a description of the functionning, not a characteristic, it shouldn't be here.
+ it's incomplete, the scan conversion isn't mentioned. Anyway, this should be in the software documentation, so just delete this paragraph please

File: stable/characteristics.md
Comment (Aurelie-Mutschler) [inline stable/characteristics.md]: This section should include the following pieces of information :
- recall the default tunings of the device --> There should be a **subsection "Default tunings"** where you recall these informations
- depth of detection, angular coverage --> the paragraph about depth of detection is not clear to me ? + these informations should be in a **subsection "Detection characteristics"**
- SNR --> can you quantify this + add the nice figures you made to visualize the last improvement? --> in a **subsection "Raw signal characteristics"**
- number of pixels, size of pixels, framerate --> can you gather those informations together with the image size, in a unique **subsection "Display characteristics"** ? 
- add a screenshot of a beautiful echOpen image :-)

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md:3]: - The link to the documentation website points to the old configuration --> @Bivi when will the website be updated with the new configuration ?

- On the repo, the config "Aquarium" is v2.1.0, whereas it's supposed to be v3.0.0

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: The synoptic diagram seems incomplete. For example, where is the high voltage ? And the WIFI ?

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: The bloc diagram is not up-to-date as it still shows the envelope detection, but it's not implemented on the device. 
Besides, I agree with @Bivi remark about the two diagrams. Let's get rid of this bloc diagram here, @clecoued and I will do a more general one that describes the device + app pipeline.
Then, the synoptic diagram should be more exhaustive (see first sentence in this comment)

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: Can you move the "### Acoustic" section to characteristics.md, in a dedicated "Transducer characteristics" section ?

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: Can you at least put a legend to describe the content of the figures ?

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: Here you mix description and characteristics. Is it possible to keep only the description in doc_hardware.md and move the signal characteristics in characteristics.md?
Anyway, this section is not clear, it should be at least more structured

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: The discussion about maximal depth should rather be in characteristics.md.
Besides, I don't understand at all the explanations given here. What's the link between sampling rate and depth of detection ?! Why do the value of the sampling rate influences the maximal depth ?!

**Review by Aurelie-Mutschler — CHANGES_REQUESTED** [review]
Here are some comments about the hardware documentation and the device characteristics.

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [reply stable/doc_hardware.md]: discussed with @jerome2echopen : let's make a diagram focused on modules. The functional diagrams (general diagrams + pipeline of data processing) will be displayed in the high-level documentation

File: stable/doc_hardware.md
Comment (Aurelie-Mutschler) [inline stable/doc_hardware.md]: as discussed, let's remove the envelope detection module ;-)

**Comment by Aurelie-Mutschler** [comment]
@jerome2echopen thanksss the doc_hardware.md is awesome !! ;-)
there's still a link to the envelope detection module that has to be dismissed. Otherwise I think we're good with doc_hardware.md
Waiting for updates of characteristics.md to merge the PR... to be continued !

**Pull request merged**
